### ISSUE TITLE
[BUG] longhorn-engine integration test test_restore_to_file_with_backing_file failed after upgrade to sles 15.4

### ISSUE THREAD

**Issue by yangchiu** [longhorn/longhorn#4632]
## Describe the bug

After [update the base image version from 15.3 to 15.4](https://github.com/longhorn/longhorn-engine/pull/752/files), the integration test case `test_restore_to_file_with_backing_file` [failed](https://drone-pr.longhorn.io/longhorn/longhorn-engine/810/3/3):

`
E               subprocess.CalledProcessError: Command '['/go/src/github.com/longhorn/longhorn-engine/integration/common/../../bin/longhorn', '--url', 'localhost:30001', '--debug', 'backup', 'restore-to-file', 's3://backupbucket@us-east-1/s3-test?backup=backup-bafda6d9969b4fcd&volume=test-8bbd430b-volume-backing', '--backing-file', '/go/src/github.com/longhorn/longhorn-engine/integration/common/../../backing_file.qcow2', '--output-file', '/go/src/github.com/longhorn/longhorn-engine/integration/common/../../test-restore_to_file-output.raw', '--output-format', 'raw']' returned non-zero exit status 1.`

`time="2022-09-28T04:22:54Z" level=fatal msg="Error running restore to file command: failed MergeSnapshotsToBackingFile backup.img.converted -> backing.img.cp: failed rebaseSnapshot backup.img.converted -> backing.img.cp: Failed to execute: qemu-img [rebase -u -b backing.img.cp backup.img.converted], output , stderr, qemu-img: Could not change the backing file to 'backing.img.cp': backing format must be specified\n, error exit status 1"`

not sure but probably related to https://gitlab.com/qemu-project/qemu/-/commit/497a30dbb065937d67f6c43af6dd78492e1d6f6d

## To Reproduce

1. update sles version from 15.3 to 15.4 like https://github.com/longhorn/longhorn-engine/pull/752/files
2. run the integration test case

## Log or Support bundle

If applicable, add the Longhorn managers' log or support bundle when the issue happens. 
You can generate a Support Bundle using the link at the footer of the Longhorn UI.

## Environment

 - Longhorn version:
 - Installation method (e.g. Rancher Catalog App/Helm/Kubectl):
 - Kubernetes distro (e.g. RKE/K3s/EKS/OpenShift) and version:
   - Number of management node in the cluster:
   - Number of worker node in the cluster:
 - Node config
   - OS type and version:
   - CPU per node:
   - Memory per node:
   - Disk type(e.g. SSD/NVMe):
   - Network bandwidth between the nodes:
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal):
 - Number of Longhorn volumes in the cluster:

**Comment by mantissahz** [comment]
Yeah, I think @yangchiu is right, it's about this commit [Require -F with -b `backing_file`](https://gitlab.com/qemu-project/qemu/-/commit/497a30dbb065937d67f6c43af6dd78492e1d6f6d)

**Comment by innobead** [comment]
@weizhe0422 this change will probably impact your PR https://github.com/longhorn/longhorn/pull/4677
